### ISSUE TITLE
Flight widget should limit to one item/row on tiny screens.

### ISSUE THREAD

**Issue by dogmaphobic** [mavlink/qgroundcontrol#3482]
iPhone 4s

Allowing more than one row simply doesn't fit. ~~The scrolling (left/right) feature is not working either. Adding another issue for that.~~

![simulator screen shot jun 5 2016 5 29 40 pm](https://cloud.githubusercontent.com/assets/749243/15808358/15fa7c12-2b44-11e6-82ae-e553050dc045.png)

**Comment by dogmaphobic** [comment]
Alright, I assumed the other "screen" (scroll right) was to show more items. I didn't realize it's something else. It's just that on mobile (even iPad) it's broken. What is this? Vibe?

![simulator screen shot jun 5 2016 5 33 56 pm](https://cloud.githubusercontent.com/assets/749243/15808428/67262d60-2b45-11e6-825d-445c8428d491.png)

**Comment by dogmaphobic** [comment]
I guess we need to rethink this widget a bit :) Even on Desktop it can be made to look quite weird. There is no limit to what you can add and it clips it. It should either limit it based on the height or allow for distribution on separate "screens".

![screen shot 2016-06-05 at 5 49 55 pm](https://cloud.githubusercontent.com/assets/749243/15808479/30d7bb1a-2b46-11e6-9d3f-fc7c3e814a0b.png)

**Comment by DonLakeFlyer** [comment]
The widget should allow you to scroll left/right to get to the different panels. Up/Down if the panel is taller than the available space.

**Comment by dogmaphobic** [comment]
Up/Down doesn't seem to be working.
What is supposed to be in the "different panels"? Now all I see is "Not Available". I selected all the options in "Values Widget Setup".
Also, if you uncheck the "Large" checkbox, it also unchecks item itself (removing it from the list).

**Comment by DonLakeFlyer** [comment]
First panel are values. Second panel is Vibe (vibration) widget. Vibe is ArduPilot only at this point. Waiting on vibration message to show up on PX4 side.

**Comment by dogmaphobic** [comment]
For what's worth, the scroll up/down is not working for me. If I have more items than can be shown, it simply gets truncated and no scrolling is possible. I first assumed it would distribute itself into several panels (the left/right scroll).

**Comment by HTRamsey** [comment]
Don't believe this is relevant anymore with that widget redesigned
